### DISCUSSION TITLE
How often do individuall targets release?

### DISCUSSION THREAD

**Discussion by laxsjo** [cssbattle/forum#287] [Q&A] [answered]
Hello, I'm currently waiting for the next target to release for Battle #20, and I'm thinking about how often they release new targets.

Assuming that battles are 30 days long (I'm not entirely sure when it started, but it currently has 25 days left), the battle should have started 5 days ago. And if we assume that the targets are released on a regular interval there should be a new target each 3.75 days (because there are 8 targets and 30 days total: 8/30 = 3.75).

Is this number correct?
Would be nice to know when new targets release.

**Comment by Beowolve** [comment, marked as answer]
In the past new targets have been released exactly after one week. So two new targets will be released at 11.08. on 13:00

**Reply by laxsjo** [reply]
Thanks for the explanaition!

**Comment by chinchang** [comment]
@Beowolve thanks Beo for that answer! 🙇🏼 

@Laxsjo 
This was a valid concern. This won't be a problem now as all locked targets will show when they'll unlock :)
![image](https://user-images.githubusercontent.com/379918/183603033-18cb88e3-8909-495a-bb05-6797a44c2e06.png)

**Reply by laxsjo** [reply]
Wow! Thank you so much :D
I did not expect this to be fixed this quickly.
Did you already have this feature planned, before reading my post?
If not I'm amazed at your quick response to suggestions!

**Reply by chinchang** [reply]
> Wow! Thank you so much :D I did not expect this to be fixed this quickly. Did you already have this feature planned, before reading my post? If not I'm amazed at your quick response to suggestions!

Nope, your question triggered it! 😄 So thank you! 🙌🏼  Since it was a quick thing to do, we did it!
